src/dialog.jsx

```diff
@@ -73,22 +66,12 @@ render() {
             <StyledButton onClick={stateManager}>
               {'🔥 Wow I am awesome 🔥'}
             </StyledButton>
-          </StyledDialog>
-        )}
+          </StyledDialog>}
       </Main>
-    );
+    )
   }
 }
 
-}
-
-
-Dialog = props => {
-  const {
-    header, message, isOpen, stateManager,completionTimeMessage
-  } = props;
```

Comment:
This was the mail culprit here ☝️

Reply:
I see, thank you for the fix, I believe I screwed things up with the browser conflict resolver. Seems like you have a conflict so if that is fixed I can merge it :)